docs/admin/kubernetes-ops/logs.md

```diff
@@ -3,17 +3,137 @@ tags:
   - administration
   - kubernetes
   - operations
+  - logs
 ---
 
-## This content is in progress
+## Overview
+The Janssen logs can be viewed using the following command:
 
-The Janssen Project documentation is currently in development. Topic pages are being created in order of broadest relevance, and this page is coming in the near future.
+```
+kubectl logs <pod-name> -n <namespace> 
+```
 
-## Have questions in the meantime?
+## Log Levels
+The following log levels can be configured through the configuration CLI:
 
-While this documentation is in progress, you can ask questions through [GitHub Discussions](https://github.com/JanssenProject/jans/discussion) or the [community chat on Gitter](https://gitter.im/JanssenProject/Lobby). Any questions you have will help determine what information our documentation should cover.
+| Log Level | Messages Logged |  
+|---------- |------------                  |  
+|Trace      | All messages                 |  
+|Debug      | Debug level and above        |  
+|Info       | Informational level and above|  
+|Warn       | Warning level and above      |  
+|Error      | Error level and above        |  
+|Fatal      | Only fatal errors            |  
+|Off        | Logging is disabled          |
 
-## Want to contribute?
+## Configuring Log Levels
 
-If you have content you'd like to contribute to this page in the meantime, you can get started with our [Contribution guide](https://docs.jans.io/head/CONTRIBUTING/).
+### auth-server
+To get the log level of current log level of auth-server:
```

Comment:
Above you can add a note that the the current log level can be found in `<helm-release-name>-config-cm` i.e `janssen-config-cm` under the `CN_<service-name>_APP_LOGGERS` as shown in the service specific entries.